proto/src/determined/api/v1/experiment.proto

```diff
@@ -817,6 +817,8 @@ message SearchExperimentsRequest {
   int32 limit = 3;
   // Sort parameters in the format <col1>=(asc|desc),<col2>=(asc|desc)
   optional string sort = 4;
+  // Filter expression
+  optional string filter = 5;
```

Comment:
I think it totally fine as a JSON string

Just curious why do we not make this into a POST request? Is it for caching?

Reply:
Yea, there are a few reasons why switching this to a `POST` request would not be a good idea. The inability to cache data is a good one. Another good example is that by using a `GET` request and query parameters on the UI we are able to better handle browser history, since the url contains the necessary parameters to re-fetch data. So it's not quite as simple as being able to just switch this api from `GET` to `POST` and everything continues to work fine in terms of UX